src/user.cc

```diff
@@ -163,9 +163,12 @@ TimeEntry *User::Start(
     const std::string project_guid,
     const std::string tags,
     const time_t started,
-    const time_t ended) {
+    const time_t ended,
+    const bool stop_current_running) {
```

Comment:
Sorry, but I dislike this change as it decreases maintainability and increases cyclomatic complexity. 
I actually don't like how the method looked previously too: having `duration`, `started`, and `ended` parameter is a confusing API. 
I think this is an opportunity to refactor this code a little. My suggestion is to create a separate method, e.g. `Create` which won't use the current time, won't stop the currently running entry. We can use that method for creating entries in manual mode as well. `Start` method can call `Create` method inside, or they can both call a common private method.

Reply:
Thank you for the proposal. 
I follow your approach by 
- Introducing the Create func: For creating a TE without stop the running TE
- Move to Start to private func (_start)
- Revert the change on Start func

Please help me to review a second round 🤚

Reply:
Initially, I've had a little different refactoring in mind, so that Start and Create functions are completely separated everywhere, in Context as well. But now I see that it's quite difficult to achieve because there would be a lot of copied code in the Context class.
The current changes don't really do any good without further refactorings, so I suggest to return to your original solution. Sorry for the confusion.

Reply:
I just reverts two last commits and it's the original implementation 🙌